extensions/container-image/container-image-podman/deployment/src/main/java/io/quarkus/container/image/podman/deployment/PodmanConfig.java

```diff
@@ -16,4 +17,10 @@ public interface PodmanConfig extends CommonConfig {
      * https://docs.podman.io/en/latest/markdown/podman-build.1.html#platform-os-arch-variant
      */
     Optional<List<String>> platform();
+
+    /**
+     * Require HTTPS and verify certificates when contacting registries
+     */
+    @WithDefault("true")
+    boolean tlsVerify();
```

Comment:
Will this parameter added automatically to the doc published here: https://quarkus.io/guides/container-image#PodmanOptions ?

Reply:
Yes